cmake/rust/union_lib.rs.in

```diff
@@ -0,0 +1,11 @@
+// This file is generated by CMake
```

Comment:
Says generated? But we commit it?

Overall this should ideally not be generated, the header part I mean should be editable and cmake concats

Reply:
Yeah it's a template ;)